<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkerWindow</class>
 <widget class="QWidget" name="MarkerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>147</y>
     <width>230</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить последнее</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>700</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Сохранить разметку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>757</width>
     <height>632</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>182</y>
     <width>230</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить все</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>60</y>
     <width>230</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Квадрат</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Треугольник</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ромб</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Линия</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>89</y>
     <width>230</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>118</y>
     <width>230</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>217</y>
     <width>230</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Востановить последнее</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
